python/tank_vendor/shotgun_authentication/login_dialog.py

```diff
@@ -159,7 +159,7 @@ def _set_login_message(self, message):
         Set the message in the dialog.
         :param message: Message to display in the dialog.
         """
-        self.ui.message.setText(message)
+        self.ui.message.setText(str(message))
```

Comment:
Hum, this might actually be a double fix. The error was that I was using e[0] to get exception state, the problem is that GeneralProxyException's e[0] is actually a tuple, which completely breaks the formatting of the message, which expects a single value and not a tuple. The exception handling in that class has always been a bit rubbish. I'll clean it up a bit more and test. This goes for all comments.